flexmeasures/cli/data_show.py

```diff
@@ -218,4 +221,117 @@ def list_data_sources():
+)
+@click.option(
+    "--duration",
+    "duration_str",
+    required=True,
+    help="Duration of the plot, after --from. Follow up with a duration in ISO 6801 format, e.g. PT1H (1 hour) or PT45M (45 minutes).",
+)
+@click.option(
+    "--belief-time-before",
+    "belief_time_before_str",
+    required=False,
+    help="Time at which beliefs had been known. Follow up with a timezone-aware datetime in ISO 6801 format.",
+)
+@click.option(
+    "--source-id",
+    "source_id",
+    required=False,
+    type=int,
+    help="Source of the beliefs (an existing source id).",
+)
+def plot_beliefs(
+    sensor_ids: List[int],
+    start_str: str,
+    duration_str: str,
+    belief_time_before_str: Optional[str],
+    source_id: Optional[int],
+):
+    """
+    Show a simple plot of belief data directly in the terminal.
+    """
+    # handle required params: sensor, start, duration
+    sensor_names: List[str] = []
+    for sensor_id in sensor_ids:
+        sensor: Sensor = Sensor.query.get(sensor_id)
+        if not sensor:
+            click.echo(f"No sensor with id {sensor_id} known.")
+            raise click.Abort
+        sensor_names.append(sensor.name)
+    start = isodate.parse_datetime(start_str)  # TODO: make sure it has a tz
+    duration = isodate.parse_duration(duration_str)
+    # handle belief time
+    belief_time_before = None
+    if belief_time_before_str:
+        belief_time_before = isodate.parse_datetime(belief_time_before_str)
+    # handle source
+    source: DataSource = None
+    if source_id:
+        source = DataSource.query.get(source_id)
+        if not source:
+            click.echo(f"No source with id {source_id} known.")
+            raise click.Abort
+    # query data
+    beliefs_by_sensor = TimedBelief.search(
+        sensors=list(sensor_ids),
+        event_starts_after=start,
+        event_ends_before=start + duration,
+        beliefs_before=belief_time_before,
+        source=source,
+        sum_multiple=False,
+    )
+    # only keep non-empty
+    beliefs_by_sensor = {
+        sensor_name: beliefs
+        for (sensor_name, beliefs) in beliefs_by_sensor.items()
+        if not beliefs.empty
+    }
+    if len(beliefs_by_sensor.keys()) == 0:
+        click.echo("No data found!")
+        raise click.Abort()
+    sensor_names = list(beliefs_by_sensor.keys())
+    first_df = list(beliefs_by_sensor.values())[0]
+
+    # Build title
+    if len(sensor_ids) == 1:
+        title = f"Beliefs for Sensor '{sensor_names[0]}' (Id {sensor_ids[0]}).\n"
+    else:
+        title = f"Beliefs for Sensor(s) [{','.join(sensor_names)}], (Id(s): [{','.join([str(sid) for sid in sensor_ids])}]).\n"
+    title += f"Data spans {naturaldelta(duration)} and starts at {start}."
+    if belief_time_before:
+        title += f"\nOnly beliefs made before: {belief_time_before}."
+    if source:
+        title += f"\nSource: {source.name}"
```

Comment:
Better use `source.description` instead, so we also get to see the model and version, if they are set.